changelog/updates/2022-12-07-libarchive-update.md

```diff
@@ -0,0 +1 @@
+- libarchive ([3.6.1](https://github.com/libarchive/libarchive/releases/tag/v3.6.1))
```

Comment:
I think we do not have to add the changelog, because libarchive is already 3.6.1 since months.

Reply:
Oh, right. That was a brainfart on my side then. I probably thought that we were still on 3.6.0. Will remove it.